packages/protocol-dashboard/src/store/cache/music/hooks.ts

```diff
@@ -37,15 +37,12 @@ export function fetchTopTracks(): ThunkAction<
 > {
   return async (dispatch, _, aud) => {
     try {
-      await aud.awaitSetup()
-      const data = (await fetchWithLibs({
-        endpoint: '/v1/tracks/trending',
-        queryParams: { limit: 4 }
-      })) as any
-      const tracks: Track[] = data.slice(0, 4).map((d: any) => ({
+      // TODO: Fix sdk type to use limit and pass 4 here
```

Comment:
This should likely be fixed before merging since the default limit for trending is much higher and we throw them all away.
Will require updating the API that generates SDK code to actually indicate it accepts a limit param 🤦